packages/generator-volto/news/5530.bugfix

```diff
@@ -0,0 +1 @@
+Fixed generator ESlint config. Added tests and docs. @sneridagh
```

Comment:
By including an error message, we can help users who are DuckDuckGo-ing for the message, and point them for how to resolve it. Yes, it's verbose, but it makes a much more enjoyable developer experience, and we know those developers can be easily irritated. 😉 

````suggestion
After creating a project using the project generator, running ESLint might return an error message in your console:

 ```console
✖ pnpm eslint --max-warnings=0 --fix:
ESLint found too many warnings (maximum: 0).
```

- Fixed the project generator's ESLint configuration.
- Added code quality checks to the CI to ensure a generated project can run these checks.
- Added documentation for how to reconfigure ESLint in existing projects.

@sneridagh
````

Reply:
@stevepiercy this is a changelog, not an issue tracker. I'm not ok with this one.

Reply:
Revised.